sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/implementation/SessionTest.java

```diff
@@ -5,6 +5,7 @@
 import com.azure.cosmos.BridgeInternal;
```

Comment:
nit: Unrelated to your PR but this file has unused imports.

Reply:
Fixed